apps/src/code-studio/components/progress/ScriptTeacherPanel.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import TeacherPanel from '../TeacherPanel';
+import ToggleGroup from '@cdo/apps/templates/ToggleGroup';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import { ViewType, setViewType, selectSection } from '../../stageLockRedux';
+import commonMsg from '@cdo/locale';
+
+const styles = {
+  viewAs: {
+    fontSize: 16,
+    margin: 10
+  },
+  toggleGroup: {
+    margin: 10
+  },
+  select: {
+    margin: 10,
+    width: 180
+  },
+  text: {
+    margin: 10
+  },
+  exclamation: {
+    color: 'red'
+  },
+  dontForget: {
+    display: 'inline',
+    marginLeft: 10,
+    fontSize: 16,
+    fontFamily: '"Gotham 7r", sans-serif'
+  }
+};
+
+const ViewAsToggle = ({viewAs, setViewType}) => (
```

Comment:
Yep, this is a functional stateless component https://facebook.github.io/react/docs/reusable-components.html#stateless-functions